esp_jpeg/examples/tjpgd/main/Kconfig.projbuild

```diff
@@ -0,0 +1,17 @@
+menu "Example Configuration"
+    config EXAMPLE_LCD_SPI_8_LINE_MODE
```

Comment:
shall we remove this since the BSP driver doesn't contain any LCD utilizes the SPI octal bus.